rotkehlchen/tests/unit/test_chains_aggregator.py

```diff
@@ -142,3 +163,62 @@ def test_detect_evm_accounts(blockchain: 'ChainsAggregator') -> None:
 
     assert set(accounts_in_db) == set(expected_accounts_data)
     assert len(accounts_in_db) == len(expected_accounts_data)
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.freeze_time('2023-06-19 05:16:10 GMT')
+@pytest.mark.parametrize('polygon_pos_manager_connect_at_start', [(ALCHEMY_POLYGON_NODE,)])
+@pytest.mark.parametrize('polygon_pos_accounts', [[make_evm_address()]])  # to connect to nodes
+def test_detect_evm_accounts_spam_tx(polygon_pos_manager: 'PolygonPOSManager') -> None:
+    """
+    Test correctly that an account with only erc20 transfers of spam tokens get correctly
+    marked as spammed.
+    The tested address has received the following spam tokens
+    eip155:137/erc20:0x91bD4023A21bc12814905f251eb348e298DBC0F0
+    eip155:137/erc20:0xD6198855979714255d711A4bB8BF1763d28A473B
+    eip155:137/erc20:0xb76c90B51338016011Eaf27C348E3D84A623C5BF
+    eip155:137/erc20:0x5522962DCE6BE2a009D29E5699a67C38b392beb9
+    eip155:137/erc20:0xd9503c336512120Aa6834Ab5d9258a32940bB2C6
+    eip155:137/erc20:0x9715A23D25399EF10D819e4999689de3d14eB7e2
+    eip155:137/erc20:0xb266edC3706fC2A48ECFef7DD8831435f12D9966
+    eip155:137/erc20:0x06732174B52743C374445E88C9b01031Bd0FB28f
+    eip155:137/erc20:0xE2Ee00F49464d6B60771dc118A1bb4eb362bd154
+    eip155:137/erc20:0x37CC5F5610d91325c8A8C0eD74d26A01F19e7B51
+
+    first we check that it is marked as NOT spammed by:
+    - ignoring the first asset
+    - adding the second as spam asset
+    - third is unknown so the address is marked as not spammed
+
+    to verify that the transaction has been spamemd all the remaining assets are ignored
```

Comment:
What do you mean by the transaction has been spammed?

```suggestion
    to verify that the transaction has been spammed all the remaining assets are ignored
```

This makes no sense in English. Do you mean, marked as spam?

Reply:
it is `the address has been spammed`